  * debian/patches/inet.diff: New patch to add hpxt1000 and latitude
    to $inetkbds.  Closes: #381882
    Thanks Steinar H. Gunderson

Submitted upstream: https://bugs.freedesktop.org/show_bug.cgi?id=7816

Index: xkb-data/rules/base.lists.part
===================================================================
--- xkb-data.orig/rules/base.lists.part
+++ xkb-data/rules/base.lists.part
@@ -27,11 +27,11 @@
               compaqeak8 compaqik7 compaqik13 compaqik18 \
               cymotionlinux \
               presario ipaq \
-              dell inspiron dellusbmm dtk2000 \
+              dell inspiron latitude dellusbmm dtk2000 \
               dexxa diamond \
               genius geniuscomfy geniuscomfy2 genius_kb19e_nb \
               emachines ennyah_dkb1008 gyration \
-              hpi6 hp2501 hp2505 hp5181 hpxe3gc hpxe3gf hpxe4xxx hpzt11xx \
+              hpi6 hp2501 hp2505 hp5181 hpxe3gc hpxe3gf hpxe4xxx hpxt1000 hpzt11xx \
               hp500fa hp5xx hp5185 \
               honeywell_euroboard \
               rapidaccess rapidaccess2 rapidaccess2a \
